addons/http_routing/models/ir_http.py

```diff
@@ -459,6 +463,7 @@ def _dispatch(cls):
                 cls._auth_method_public()
         except Exception as e:
             return cls._handle_exception(e)
+        cls._dispatch_post_auth_hook()
```

Comment:
Can't think of a better way than this hook
I'd have prefer to do that in `_add_dispatch_parameter` but this is called only on is_frontend requests :/ that would also have allow to directly call request.redirect

Reply:
in which case we would like to support fw for a non is_frontend route ? 
Reminder, this option is only useful during dev. Not really in prod... 
In worst case, it is acceptable for me to switch to / instead of current page if you need to be sure that the switch is done on a is_frontend route.

Reply:
As discuss in pm, let try to move it in external controller. ty

Reply:
> in which case we would like to support fw for a non is_frontend route ? [..] if you need to be sure that the switch is done on a is_frontend route.

You didn't understand, that's the opposite, I'd like that behavior to only happen on frontend route, but in the current PR state, we don't know yet if we are in frontend route at that point in the dispatch.
So can't put the second part of the code in add_dispatch_parameter as only called in frontend route, so I created a hook that is called for every route

But I will push in a few minutes a diff to use a route instead as discussed, that's a bigger diff that you imagine IMO, there is a branch for design-theme, and if we drop the `fw` support in 13.0, also in upgrade repo (for tests).

If we want to keep the retrocompatibility in 13.0 for FW, the diff will still have to be applied in master even in design theme, but I don't think we should keep retro here